File: MODUL 4 PEMROGRAMAN 1/PRAK401-2310817120002-SITI RATNA DWINTA SARI.py
for i in range(3):
    klp, simbol = input().split()
    klp = int(klp)
    for j in range(1, 51):
        if j % klp == 0:
            print(simbol, end=" ")
        else:
            print(j, end=" ")
    print()

File: MODUL 4 PEMROGRAMAN 1/PRAK401-2310817120002-SITIRATNADWINTASARI.c
#include <stdio.h>
int main (){
    int klp;
    char simbol;
    for (int i = 1; i <= 3; i++){
    scanf("%d %c", &klp, &simbol);
    for (int i = 1; i <= 50; i++){
        if (i % klp == 0){
            printf("%c ", simbol);
        } else{
            printf("%d ", i);
        }
    }
    } return 0;
}

